### ISSUE TITLE
Add hidden text voor the external link icon

### ISSUE THREAD

**Issue by Veyfeyken** [a-ui/core_branding_scss#148]
### Problem

[atoms/atoms.link](https://a-ui.github.io/core_branding_scss/#atoms-link)

The external link icon has no text-alternative. It's not decorative, a screenreader user cannot know the link is external. 

```html
<a href="#" class="has-icon-right">
  This is a nice link with an icon to the right<span class="fa fa-external-link"></span>
</a>
```

![20201109-09-03-04](https://user-images.githubusercontent.com/870049/98515659-aa5d0500-226b-11eb-871c-a44b8bd761b0.jpg)

### Solution

Add hidden text for screenreader users.

Suggestion:

```html
<a href="#" class="has-icon-right">
  This is a nice link with an icon to the right
  <span class="fa fa-external-link"></span>
  <span class="u-screen-reader-only">(external)</span>
</a>
```

The ACPaas UI documentation and examples are in English. It's highly important that all default hidden text and alt-attributes are translated to Dutch when they are implemented. Maybe add a reminder to all instances in the documentation?

### Links and articles

[Understanding Success Criterion 1.1.1: Non-text Content
](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)

**Comment by TriangleJuice** [comment]
Fixed in #178
